sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/AnalyzeActionsAsyncClient.java

```diff
@@ -93,9 +100,9 @@
     private static final String ENTITY_RECOGNITION_PII_TASKS = "entityRecognitionPiiTasks";
     private static final String KEY_PHRASE_EXTRACTION_TASKS = "keyPhraseExtractionTasks";
     private static final String ENTITY_LINKING_TASKS = "entityLinkingTasks";
-
-    private static final String REGEX_ACTION_ERROR_TARGET =
-        "#/tasks/(keyPhraseExtractionTasks|entityRecognitionPiiTasks|entityRecognitionTasks)/(\\d+)";
+    private static final String SENTIMENT_ANALYSIS_TASKS = "sentimentAnalysisTasks";
+    private static final String REGEX_ACTION_ERROR_TARGET = "#/tasks/(keyPhraseExtractionTasks|"
```

Comment:
Make this into a static `Pattern`. Right now this is being compile in a potential tight loop which is very costly.